src/list.html

```diff
@@ -95,6 +95,16 @@ <h2 class="subtitle">
             <ul>
               <li><a href="https://reichrobert.typeform.com/to/V0WnZf" target="_blank">NewTech Colorado</a> - New Tech is a community of 12,000 geeks who live and work in Colorado.</li>
             </ul>
+          <ul>
```

Comment:
This is an extra `ul`.
Take a look at the list on the preview site: https://deploy-preview-56--telephone-operator-bonnie-31287.netlify.com/list.html